<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Determinants</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTableWidget" name="detTable">
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string notr="true">New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>1</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QTableWidget" name="scoreTable">
      <row>
       <property name="text">
        <string>Игрок 1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Игрок 2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Счет</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>0</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="moveLabel">
        <property name="text">
         <string>Ход игрока 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Ход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Определитель:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="debugLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDeterminants">
    <property name="title">
     <string>Determinants</string>
    </property>
   </widget>
   <addaction name="menuDeterminants"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
